src/components/InfoBar.js

```diff
@@ -0,0 +1,18 @@
+import React, { useState} from 'react'
+import { FaInfoCircle } from "react-icons/fa"
+
+const InfoBar = () => {
+  const [ hover, setHover ] = useState(false)
+
+  return (
+    <div
+      onMouseEnter={() => setHover(true)}
+      onMouseLeave={() => setHover(false)}
+      className='infobar'
```

Comment:
This is not accessible 😭 😭 😭  This would only show for mouse users, not keyboard, mobile, or screen reader users. I recommend using this tooltip: https://reach.tech/tooltip/